providers/redis/types.go

```diff
@@ -67,25 +70,26 @@ type Config struct {
 	// new connection is slow.
 	MinIdleConns int
 
-	// Connection age at which client retires (closes) the connection.
-	// Default is to not close aged connections.
-	MaxConnAge time.Duration
```

Comment:
Please don't remove this fields to not breaks compatibility without increase major version. So keep it, add a deprecated notice, and use it if not empty.

Reply:
I've kept these in as deprecated fields and remapped their values to the new fields where they still exist/have been renamed.